### ISSUE TITLE
Launch Pad Digitizer FrackFinder App on Crowd.SkyTruth.org

### ISSUE THREAD

**Issue by ezys** [SkyTruth/CrowdProjects#24]
Depends on #23

- [x] Launch a clean app to Crowd.SkyTruth.org for the Frack-A-Thon (All active wellpads identified in previous FrackFinder projects, redundancy set at 5 or 10, your discretion.) 
- [x] Set as "featured" application on crowd.skytruth.org

**Comment by geowurster** [comment]
Example command:

```
./createTasks.py \
    -c \
    -n 3 \
    -s http://crowd.dev.skytruth.org \
    -k 49e1ae77-5cf9-423e-81b4-2e8a3950abde \
    -a digitizer-pad/ \
    -r k-test5 \
    -t ~/github/CrowdProjects/Data/FrackFinder/OH/2010-2013/Pad-Delineator/input_tasks/input_tasks.json
```

**Comment by geowurster** [comment]
@ezys Need a name.

**Comment by ezys** [comment]
FrackFinder Ohio: Wellpad Tracer

**Comment by Yolandeezy** [comment]
it's featured --> http://crowd.skytruth.org/

**Comment by geowurster** [comment]
@ezys Looks like the tutorial isn't working but you will be in the room with them so to get to a new task you can just do `http://crowd.skytruth.org/app/frackfinder-ohio-wellpad-tracer/task/newtask`

I created a ticket for @Yolandeezy to fix it - https://github.com/SkyTruth/CrowdProjects/issues/27.

**Comment by ezys** [comment]
Above link is incorrect. This works...

http://crowd.skytruth.org/app/frackfinder-ohio-wellpad-tracer/newtask
